libraries/featureflag/api/src/main/kotlin/io/element/android/libraries/featureflag/api/FeatureFlagService.kt

```diff
@@ -33,3 +36,6 @@ interface FeatureFlagService {
      */
     suspend fun setFeatureEnabled(feature: Feature, enabled: Boolean): Boolean
 }
+
+suspend fun FeatureFlagService.isFeatureEnabled(feature: Feature): Boolean =
```

Comment:
This makes it a breaking change.
If you kept this API as it was and added the flow variant to the interface, it would be an "addition only change" and hence non-breaking.

Reply:
You're right, I have updated the PR.